package.json

```diff
@@ -24,15 +25,16 @@
   "devDependencies": {
     "dotenv": "^16.0.0",
     "jest": "^27.5.1",
-    "serverless": "^3.35.2",
+    "serverless": "3.39.0",
```

Comment:
Also, there is a version 3.40.0 if you want to use that.

```suggestion
    "serverless": "^3.39.0",
```

Reply:
Good catch